Move ascii arts to their own file to avoid bloat (#39847)
CleverRaven/Cataclysm-DDA: doc/JSON_INFO.md

## `data/json/` JSONs

### Ascii_arts

| Identifier        | Description
|---                |---
| id                | Unique ID. Must be one continuous word, use underscores if necessary.

> | picture           | Array of string, each entry is a line of an ascii picture and must be less than 42 characters long.

Reviewer: ```suggestion
| picture           | Array of string, each entry is a line of an ascii picture and must be at most 42 columns long.
```
